### ISSUE TITLE
Maximize button does not work

### ISSUE THREAD

**Issue by amnweb** [drcika/apc-extension#109]
Maximize button does not work if transparent is **true**

```
Version: 1.84.2 (user setup)
Commit: 1a5daa3a0231a0fbba4f14db7ec463cf99d7768e
Date: 2023-11-09T10:51:52.184Z
Electron: 25.9.2
ElectronBuildId: 24603566
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.22631
```

![Screenshot 2023-11-29 002048](https://github.com/drcika/apc-extension/assets/16545063/3d01e0f5-e53f-4186-b979-117b403bd79c)

**Comment by bbilginerdem** [comment]
```
"apc.electron": {
		"frame": false,
		"titleBarStyle": "hiddenInset",
		"window.titleBarStyle": "native",
		"vibrancy": "ultra-dark",
		"opacity": 0.98,
		"visualEffectState": "active",
    "transparent": true
	},
```

using these settings given in the demo folder, it works for me. I was doing something similar to what you did as well @amnweb

**Comment by amnweb** [comment]
> using these settings given in the demo folder, it works for me. I was doing something similar to what you did as well @amnweb

No, on Windows `hiddenInset` won’t show `titleBarOverlay`. It looks like a bug or some kind of electron limitation. I’m waiting for @drcika to confirm this.

This is actually what I want to make (just an example), but maximize doesn’t work.

![Screenshot 2023-12-21 021234](https://github.com/drcika/apc-extension/assets/16545063/8776ce77-4d67-4399-b159-ab056117970a)

**Comment by Neix20** [comment]
Can confirm, If you toggle zen mode, it will auto hide the maximize & minimize button
I need to reload my vscode to show button again
```
Version: 1.85.1 (user setup)
Commit: 0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2
Date: 2023-12-13T09:49:37.021Z
Electron: 25.9.7
ElectronBuildId: 25551756
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.19045
```
[apc_issue_ii.webm](https://github.com/drcika/apc-extension/assets/66442152/8d982613-43c7-4e72-b3d7-15ea2cf62c06)
